src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -88,6 +88,13 @@ func (r *ClusterReconciler) Reconcile(
 		return ctrl.Result{}, fmt.Errorf("unable to retrieve Cluster resource: %w", err)
 	}
 
+	managedAnnotationKey := redpandav1alpha1.GroupVersion.Group + "/managed"
+	if managed, exists := redpandaCluster.Annotations[managedAnnotationKey]; exists && managed == "false" {
+		log.Info(fmt.Sprintf("management of %s is disabled; edit '%s' to enable",
```

Comment:
NIT: the log saying `edit to enable` is not actionable.
```suggestion
		log.Info(fmt.Sprintf("management of %s is disabled; change %s annotation to true or remove",
```

Reply:
`or remove it`

Reply:
I updated the message.